src/components/Date/DateField.js

```diff
@@ -33,8 +44,9 @@ const DateField = ({ date, intl, relative = true, capitalize = false }: Props) =
         return Message;
     }
 
-    const formattedDate = intl.formatDate(d, { weekday: 'short', month: 'short', year: 'numeric', day: 'numeric' });
-    return <span>{formattedDate.replace(/,/g, '')}</span>;
+    let formattedDate = intl.formatDate(d, dateFormat);
+    formattedDate = omitCommas ? formattedDate.replace(/,/g, '') : formattedDate;
+    return <span>{formattedDate}</span>;
```

Comment:
Try just returning formattedDate here and see if that looks fine.
Span probably not needed here.

Reply:
Looks fine without the span, checked in the explorer w/ relative dates as well.